app/Models/Order.php

```diff
@@ -5,19 +5,38 @@
 use Illuminate\Database\Eloquent\Factories\HasFactory;
 use Illuminate\Database\Eloquent\Model;
 use Illuminate\Database\Eloquent\Relations\BelongsTo;
+use Illuminate\Database\Eloquent\Relations\HasOne;
 use Illuminate\Database\Eloquent\Relations\HasMany;
 
 class Order extends Model
 {
     use HasFactory;
 
-    public function orderLines(): HasMany
+    protected $fillable = ['id', 'order_date', 'email', 'lid_name', 'order_status_id', 'user_id', 'group_id'];
+
+    public function getRouteKeyName(){
+        return "id";
+    }
```

Comment:
Deze functie wordt nergens gebruikt.

Reply:
Is wel handig om duidelijk te hebben wat de route key is, maar hij zou weg kunnen